sympy/printing/fcode.py

```diff
@@ -55,15 +57,25 @@ class FCodePrinter(CodePrinter):
     printmethod = "_fcode"
     language = "Fortran"
 
+    type_mappings = {
+        Type('intc'): ('integer(c_int)', {'iso_c_binding': 'c_int'}),
+        float32: ('real(4)', None),
+        float64: ('real(8)', None),
+        complex64: ('complex(4)', None),
+        complex128: ('complex(8)', None),
+        Type('integer'): ('integer', None),
+        Type('bool'): ('logical', None)
+    }
+
     _default_settings = {
         'order': None,
         'full_prec': 'auto',
-        'precision': 15,
+        'precision': 17,
```

Comment:
Why did you change this? Assumedly 17 was used so that you always get full precision.